code/modules/mob/living/carbon/carbon.dm

```diff
@@ -155,7 +155,14 @@
 			return TRUE
 
 	if(thrown_thing)
-		visible_message("<span class='danger'>[src] throws [thrown_thing].</span>", \
+		if(prob(1))
+			visible_message("<span class='hierosay big'>YEET!</span>")
+			visible_message("<span class='danger'>[src] YEETS [thrown_thing].</span>", \
+						"<span class='danger'>You YEET [thrown_thing].</span>")
```

Comment:
Someone will remove this when the word 'YEET' will inevitably be added to the word filter, I just know it!